src/components/NavMenu/NavMenu.jsx

```diff
@@ -1,91 +1,107 @@
 /* eslint-disable react/require-default-props */
 import React, { useState, createElement } from 'react';
-import { Link, useLocation } from 'react-router-dom';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
 import { Menu, Layout, Button } from 'antd';
-import PropTypes from 'prop-types';
+import { instanceOf } from 'prop-types';
 import {
   ProfileOutlined,
   TableOutlined,
   DashboardOutlined,
   FormOutlined,
   MenuUnfoldOutlined,
   MenuFoldOutlined,
+  LogoutOutlined,
+  UserOutlined,
 } from '@ant-design/icons';
 import Logo from '../../assets/Logo.png';
 import styles from './NavMenu.module.css';
 import './NavAnt.css';
 
+import { Cookies, withCookies, cookieKeys } from '../../util/cookie_utils';
+import { AUTH_ROLES, logout } from '../../util/auth_utils';
+
 const { Sider } = Layout;
 
-const NavMenu = ({ isAdmin }) => {
+const NavMenu = ({ cookies }) => {
   const [collapsed, setCollapsed] = useState(false);
   const [menuTitle, setMenuTitle] = useState('Abound Food Care');
-  const location = useLocation();
+  const { pathname } = useLocation();
 
   const handleToggle = () => {
     setCollapsed(!collapsed);
     setMenuTitle(collapsed ? 'Abound Food Care' : '');
   };
 
+  const navigate = useNavigate();
+
   return (
-    <Sider trigger={null} collapsible collapsed={collapsed} className={styles['side-nav']}>
-      <Link to="/">
-        <div className={`${collapsed ? styles.collapsed : ''} ${styles.logo}`}>
-          <img src={Logo} className={styles['logo-img']} alt="AFC Logo" />
-          {menuTitle}
-        </div>
-      </Link>
-      <Menu defaultSelectedKeys={[location.pathname]} mode="inline" className={styles.menu}>
-        <Menu.Item className={styles['menu-item']} key="/" icon={<DashboardOutlined />}>
-          <Link to="/admin" className={styles.link}>
-            Dashboard
-          </Link>
-        </Menu.Item>
-        <Menu.Item className={styles['menu-item']} key="/events" icon={<FormOutlined />}>
-          <Link to="/events" className={styles.link}>
-            Events
+    <>
+      {pathname !== '/auth' && pathname !== '/reset-password' && (
```

Comment:
alright if you're gonna check for multiple paths to not render this in you might as well make an array outside of the function and check that your pathname isn't in that array of paths to not render on. this is what i get for reviewing files not in order 😩